docs/release-notes.md

```diff
@@ -1,6 +1,18 @@
 # GA Releases
 
-This page summarizes changes in each major update in Timeplus, including new features and important bug fixes.
+This page summarizes changes for each major update in Proton and Timeplus Cloud, including new features and important bug fixes.
+
+## Oct 16, 2023
+
+*Proton:*
```

Comment:
Proton v1.3.15 adds the capabilities to create/delete user-defined-function(UDF) with SQL (me, Jove, will update the docs in next 3 hours)